pom.xml

```diff
@@ -637,6 +668,17 @@
   </dependencyManagement>
 
   <dependencies>
+    <!-- Required dependency for rest-client-jakarta -->
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>dto</artifactId>
+      <classifier>patch-builders-jakarta</classifier>
+    </dependency>
+    <!-- Required dependency for rest-client-jakarta -->
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
+    </dependency>
```

Comment:
Do you need these two among the dependencies? isn't it enough that it's in dependency management?

Reply:
No, because these are excluded on line 397.

Reply:
I had to revert the jakarta versions, because it breaks the CLI. I will commit part of this patch though.